contracts/PolygonZkEVMBridge.sol

```diff
@@ -587,35 +578,29 @@ contract PolygonZkEVMBridge is
      * @param index Index
      */
     function isClaimed(uint256 index) public view returns (bool) {
-        uint256 claimedWordIndex = index / 256;
-        uint256 claimedBitIndex = index % 256;
-        uint256 claimedWord = claimedBitMap[claimedWordIndex];
-        uint256 mask = (1 << claimedBitIndex);
-        return (claimedWord & mask) == mask;
+        (uint256 wordPos, uint256 bitPos) = _bitmapPositions(index);
```

Comment:
maybe is worth it to take a look at https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/structs/BitMaps.sol

Reply:
as pointed by @invocamanman, `_setAndCheck` could be added in oppenzeppelin library